README.md

```diff
@@ -1,24 +1,43 @@
 # Adobe Experience Platform - Media extension for Android
 
-[![CircleCI](https://dl.circleci.com/status-badge/img/gh/adobe/aepsdk-media-android/tree/dev-v2%2E0%2E0.svg?style=svg)](https://dl.circleci.com/status-badge/redirect/gh/adobe/aepsdk-media-android/tree/dev-v2%2E0%2E0)
-[![codecov](https://codecov.io/gh/adobe/aepsdk-media-android/branch/main/graph/badge.svg?token=UgtU9vDA6j)](https://codecov.io/gh/adobe/aepsdk-media-android)
+[![Maven Central](https://img.shields.io/maven-central/v/com.adobe.marketing.mobile/media.svg?logo=android&logoColor=white&label=media)](https://mvnrepository.com/artifact/com.adobe.marketing.mobile/media)
```

Comment:
Should we add coverage and build tags?

Reply:
No. We decided not to add those in Readme.md